webui/react/src/components/UPlotChart/tooltipsPlugin.ts

```diff
@@ -115,25 +112,29 @@ export const tooltipsPlugin = ({ getXTooltipHeader, getXTooltipYLabels }: Props
         barEl = document.createElement('div');
         barEl.className = css.bar;
         uPlot.root.querySelector('.u-over')?.appendChild(barEl);
-
       },
       setCursor: (uPlot: uPlot) => {
         const { left, idx, top } = uPlot.cursor;
 
-        if (
-          (idx == null && displayedIdx)
-          || !left || left < 0
-          || !top || top < 0
-        ) {
-          hide();
+        if (!left || left < 0 || !top || top < 0 || idx == null) {
+          if (displayedIdx) hide();
           return;
         }
 
-        if (idx != null && idx !== displayedIdx) {
-          showIdx(uPlot, idx);
+        if (idx !== displayedIdx) {
+          const hasXValue = !!uPlot.series.find((serie, serieId) => (
```

Comment:
i pushed the fix we discussed yesterday here (not showing the line - and the tooltip - when the tooltip would be empty).

can test on [this page ](http://localhost:3000/det/experiments/330/trials/381)with just "accuracy" turned on.

Reply:
that worked beautifully!